service/history/execution/context.go

```diff
@@ -368,12 +369,12 @@ func (c *contextImpl) CreateWorkflowExecution(
 	resp, err := c.createWorkflowExecutionWithRetry(ctx, createRequest)
 	if err != nil {
 		if c.isPersistenceTimeoutError(err) {
-			c.notifyTasksFromWorkflowSnapshot(newWorkflow, true)
+			c.notifyTasksFromWorkflowSnapshot(newWorkflow, events.PersistedBlobs{persistedHistory}, true)
```

Comment:
Notification are not call when there is an error but it is not isPersistenceTimeoutError.
This can probably be simplified, but I will leave as is for now.

Reply:
yes, missed that return. thanks